examples/CMakeLists.txt

```diff
@@ -1,8 +1,63 @@
 cmake_minimum_required (VERSION 3.2)
 
+# added by JDH
+set(sqlite_orm_examples_VERSION "1.8.0")
+set(PACKAGE_VERSION ${sqlite_orm_examples_VERSION})
+
+project("sqlite_orm_examples" VERSION ${PACKAGE_VERSION})
+
+# Handling C++ standard version to use
+option(SQLITE_ORM_ENABLE_CXX_20 "Enable C++ 20" OFF)
+option(SQLITE_ORM_ENABLE_CXX_17 "Enable C++ 17" OFF)
```

Comment:
yes this is to be called like this from the command line:

`cmake  -G"Visual Studio 17 2022"   -Ax64  -DSQLITE_ORM_ENABLE_CXX_20=ON     -DSQLite3_INCLUDE_DIR=C:\Components\vcpkg\installed\x64-windows\include  -DSQLite3_LIBRARY=C:\Components\vcpkg\installed\x64-windows\lib\sqlite3.lib    -S . -B _binary2022plain_`

for instance for VS2022 and C++ 20

and for VS2015 it would be invoked like this:

`cmake  -G"Visual Studio 14 2015 Win64"   -DSQLite3_INCLUDE_DIR=C:\Components\vcpkg\installed\x64-windows\include  -DSQLite3_LIBRARY=C:\Components\vcpkg\installed\x64-windows\lib\sqlite3.lib    -S . -B _binary2015plain_`

so yes I think this way we can have the projects under one solution (that is Visual Studio solution) just like the unit tests

Reply:
which is something that is lacking!!

Reply:
@trueqbit I need your review here cause I am not a pro in CMake